static_src/test/functional/expressive_empty_state.spec.js

```diff
@@ -0,0 +1,40 @@
+
+import http from 'axios';
+import dedent from 'dedent';
+
+import EmptyElement from './pageobjects/empty.element';
+import OrgQuicklookElement from './pageobjects/org_quicklook.element';
+
+describe('Expressive empty state', function () {
+  let appsEmptyState;
+
+  it('navigates to page', function () {
+    browser.url('/');
+  });
+
+  it('changes to app emtpy state', function(done) {
+    const server = browser.getUrl().split('/')[2];
+
+    const result = browser.executeAsync(function(url, done) {
+      var req = fetch(`http://${url}/_admin/api/route/space-summary`, {
+        method: 'POST',
+        body: { "variant": "space-summary-empty-state" }
+      })
+      .then((res) => { done({msg: 'success', data: res}) })
+      .catch((err) => { done({msg: 'error', data: err, req: req}) });
+    }, server);
+
+    console.log('result', result);
+    console.log(result.value);
+  });
+
+  describe('for apps on overview', function() {
+    beforeEach(function () {
+      appsEmptyState = new EmptyElement( browser, EmptyElement.primarySelector);
```

Comment:
FYI, `BaseElement` doesn't take selector strings as an argument, you need to wrap this with `browser.element(EmptyElement.primarySelector)`.

BUT I added support for selectors in https://github.com/18F/cg-dashboard/pull/1056 because it's a good idea :D